src/main/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundQueueConsumerService.java

```diff
@@ -0,0 +1,70 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.nhs.digital.nhsconnect.lab.results.inbound.queue.DataToSend;
+import uk.nhs.digital.nhsconnect.lab.results.mesh.message.InboundMeshMessage;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Interchange;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.InterchangeHeader;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Transaction;
+import uk.nhs.digital.nhsconnect.lab.results.outbound.queue.GpOutboundQueueService;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
Should we use the new way of doing it? Plus to make it consistent across other classes as well

```suggestion
@RequiredArgsConstructor(onConstructor_ = @Autowired)
```

Reply:
Resolving, because already has been mentioned by Andrew